### DISCUSSION TITLE
V3 compatibility?

### DISCUSSION THREAD

**Discussion by tonypartridge** [saade/filament-fullcalendar#86] [General]
Any plans for Filament V3 yet?

**Comment by saade** [comment]
Hello friends.

Just released v3.0.0-beta1.

PLEASE READ CAREFULLY:

- This is a first-pass release for supporting v3.
- There WILL BE BRAKING CHANGES when the final v3.0.0 is released.
- The docs are not updated yet. But there are no braking changes for now.


Thanks for everyone who opened a PR for supporting v3. :yellow_heart:
